docs/tutorials/aws/AIConnectorHelper.ipynb

```diff
@@ -37,9 +37,22 @@
     "            domain_info = response['DomainStatus']\n",
     "\n",
     "            # Extract the domain URL and ARN\n",
-    "            domain_url = domain_info['Endpoint']\n",
     "            domain_arn = domain_info['ARN']\n",
     "\n",
+    "            # Check if domain has VPC endpoints\n",
+    "            if 'Endpoints' in domain_info:\n",
+    "                # VPC domain case\n",
+    "                if 'vpc' in domain_info['Endpoints']:\n",
```

Comment:
So for VPC endpoints there are multiple endpoints? Trying to understand the `Endpoints` objects. What else does this contain?

Reply:
It returned a Map including 'vpc':'vpc-endpoint-h2dsd34efgyghrtguk5gt6j2foh4.us-east-1.es.amazonaws.com'.